### PR TITLE
Move navigation column to right of page

### LINKED ISSUE #281: Move navigation column to the right
It was mentioned in gitter chat sometime ago that the navigation column should be on the right.

something like this?
screen shot 2016-11-09 at 10 02 23 pm

### PR DESCRIPTION
**Pull request by Mariatta**
Closes #281

### CODE REVIEW

File: _static/custom.css [hunk 1]
```diff
@@ -16,4 +16,11 @@ img.logo {
 
 .sphinxsidebarwrapper a.current.reference.internal {
     font-weight: bold;
+}
+
+div.bodywrapper {
+    margin: 0 220px 0 0;
+}
+div.sphinxsidebar {
+    float: right;
```
Comment (Adrianzatreanu) [inline _static/custom.css]: you need a newlien at the end of the file :)

**Comment by Adrianzatreanu** [comment]
this looks good, but because its a design issue, i wont ack it, but instead give it a thumbs up so other people can see it too :)

**Comment by sils** [comment]
@Mariatta how does it look?

**Comment by sils** [comment]
we usually want the *full issue url* after the closes, not just the number, because the number isn't linked when browsing the commits in the CLI locally

File: _static/custom.css (hunk 1 above)
Comment (Mariatta) [inline _static/custom.css]: Will adjust this later today :) Thanks

**Comment by Mariatta** [comment]
@sils this is the screenshot
At least I think this is what you had in mind.
If not just let me know.
Thanks.

376f9904-a6c8-11e6-8e76-633860b52789

**Comment by sils** [comment]
@Mariatta what's your opinion on the design? How should we do it?

**Comment by Mariatta** [comment]
I think I rebased properly, please check :)

**Comment by Adrianzatreanu** [comment]
nope, still out of date :D use cobot explain rebase on gitter :)

**Comment by Mariatta** [comment]
ok I think this is rebased now?

**Comment by Adrianzatreanu** [comment]
apparently theres a conflicting file or something, someone modified _static/custom.css in the mean time, or you did something wrong with the rebase, im not 100% sure tbh.

also, i kind of changed my mind, i came to realize that having the navigation column on the left side is more intuitively to click, moving it to the right side would just require me to use more brain to search there ;d i vote for staying with it on the left side ...

**Comment by Mariatta** [comment]
I'm ok with keeping the nav on the left. If maintainers agree, feel free to just close this PR anytime :)

**Comment by sils** [comment]
ok

**Pull request closed without merging**